### ISSUE TITLE
about figure 1

### ISSUE THREAD

**Issue by oukohou** [htyjers/NTN-Diff#1]
Great work!
but:
<img width="1738" height="1198" alt="Image" src="https://github.com/user-attachments/assets/053b5cfd-4453-4856-acba-0d5ce5dbb0d5" />

1. Regarding the three rows of images in Figure 1, how do they respectively illustrate the issues of: (a) Semantics inconsistency between masked and unmasked regions, and (b) Unmasked regions being disrupted due to text prompts?

2. The red dashed line represents the Ground Truth (GT). Why does the Euclidean distance begin to increase after 70 steps?

3. How do these three rows of images collectively demonstrate the superior performance of the NTN method?

Additionally, will the training code be made publicly available?

**Comment by htyjers** [comment]
1. We compare the differences between the baseline (red dashed line) and the values (ranging from purple to yellow) and we have illustrated this in the introduction. Denoised Image-Text Distance represents the semantics consistency and the Denoised Image-GT Distance represents the the preservation for unmasked regions.

2. The variation is related to the noise level, as the red dashed line indicates the distance between the ground truth and the text prompt across different timesteps.

3. We implement a training-free framework.

**Comment by oukohou** [comment]
thanks for the timely reply.
